apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx

```diff
@@ -50,13 +50,20 @@ const getTooltipContent = (breakpoint: Breakpoint) => {
   }
 };
 
+// Source https://github.com/tombigel/detect-zoom/blob/master/detect-zoom.js
+// Won't work with devtools open
+// Won't work in firefox, workaround is too crazy
+const detectZoom = () => {
```

Comment:
May be this is better?
`2 * window.devicePixelRatio !== Math.floor(2 * window.devicePixelRatio)`

Reply:
won't work anyway with 50% 200% 75%